packages/simon/src/frontleft.js

```diff
@@ -15,9 +15,13 @@ export default (part) => {
     })
   }
 
-  return options.buttonholePlacketStyle === 'seamless'
-    ? frontLeftSeamless(part)
-    : options.seperateButtonholePlacket
-    ? frontLeftClassicSeperate(part)
-    : frontLeftClassicCuton(part)
+  if (options.seperateButtonholePlacket) {
+    return frontLeftClassicSeperate(part)
+  } else if (options.buttonholePlacketStyle === 'seamless') {
+    return frontLeftSeamless(part)
+  } else if (options.buttonholePlacketStyle === 'classic') {
+    return frontLeftClassicCuton(part)
+  } else {
+    throw `Unexpected buttonholePlacketStyle: ${options.buttonholePlacketStyle}`
+  }
```

Comment:
I thought this part deserved to be a bit more verbose. But yeah, I think I tend toward that. I also really wanted to throw otherwise and that didn't work the first time I tried with the ternary. Maybe I just needed some parens.